src/screens/MessageDetails.js

```diff
@@ -92,23 +92,23 @@ export class MessageDetailsView extends React.PureComponent {
         <Text style={styles.topTitle}>SIGN MESSAGE</Text>
         <Text style={styles.title}>FROM ACCOUNT</Text>
         <AccountCard
-          title={this.props.sender.name}
-          address={this.props.sender.address}
-          networkKey={this.props.sender.networkKey}
+          title={sender.name}
+          address={sender.address}
+          networkKey={sender.networkKey}
           onPress={() => {
-            this.props.onPressAccount(this.props.sender);
+            onPressAccount(sender);
           }}
         />
         <Text style={styles.title}>MESSAGE</Text>
         <Text style={styles.message}>
-          {isAscii(this.props.message)
-            ? hexToAscii(this.props.message)
-            : this.props.data}
+          {isAscii(message)
+            ? hexToAscii(message)
+            : data}
```

Comment:
i'm confused here. If message is Ascii, treat is as a hex and do a conversion on it to Ascii, otherwise show the raw hex data? Seems like this bit has been around since pre v2.2 so I'm surprised it hasn't been caught but should it be 
```
isAscii(message) ? message : hexToAscii(data)
```
?

Reply:
yup, it def has been around for a while. Never tried to signed data that's not a Ascii though.
you suggest `hexToAscii(data)`, but that wouldn't look good if the hex isn't hex encoded ascii right? why not `isAscii(message) ? message : data` ?

Reply:
that makes more sense